ignite-generator/component/templates/component-test.js.template

```diff
@@ -0,0 +1,24 @@
+import test from 'ava'
+import React from 'react'
+import { shallow } from 'enzyme'
+import <%= name %> from '../../App/Components/<%= name %>'
+
+test('component exists', t => {
+  const wrapper = shallow(<<%= name %>  />)
```

Comment:
All good!  It's more picky than anything.  I usually make a vanilla wrapper up top.  This is for testing basics of the component.  Should I want to test differences I'll make a second const inside each test.   But in some cases 3 tests all use the same wrapper.   Might just be me, I'm flexible on this.